src/main/java/org/folio/helper/BindHelper.java

```diff
@@ -71,10 +82,25 @@ private Future<ReceivingResults> processBindPieces(BindPiecesCollection bindPiec
       .map(piecesGroupedByPoLine -> prepareResponseBody(piecesGroupedByPoLine, bindPiecesCollection));
   }
 
+  private Future<Map<String, List<Piece>>> checkRequestsForPieceItems(Map<String, List<Piece>> piecesGroupedByPoLine, BindPiecesCollection bindPiecesCollection, RequestContext requestContext) {
+    var itemIds = extractAllPieces(piecesGroupedByPoLine)
+      .map(Piece::getItemId)
+      .filter(StringUtils::isNotEmpty)
+      .toList();
+
+    return inventoryItemRequestService.getItemsWithActiveRequests(itemIds, requestContext)
```

Comment:
items that can be located in other tenants: For example item1 in tenantA, item2 in tenantB
Earch piece has receivingTenantId that we can use to fetch items from correct tenant.
Basically we did not have requirement for R to transfer requests across tenants. So for now we can add validation if pieces have differrent Piece::receivingTenantId - throw an error

Reply:
Added another validation step

Reply:
Should this validation happen in which cases:
1) When requests are found and requestAction is TRANSFER
2) When requests are found and requestAction is not TRANSFER
3) When requests are not found

I am thinking only the 1st case should be applicable for this validation

Reply:
I think all actions except Do nothing.
Even if Cancel action is not used now, but we dont support it in multitenant env now, so should throw an error

Reply:
Okay then should we validate it when requests are found or not?